web/api/v1/api.go

```diff
@@ -108,6 +108,31 @@ type rulesRetriever interface {
 	AlertingRules() []*rules.AlertingRule
 }
 
+// PrometheusVersion contains build information about Prometheus.
+type PrometheusVersion struct {
+	Version   string `json:"version"`
+	Revision  string `json:"revision"`
+	Branch    string `json:"branch"`
+	BuildUser string `json:"buildUser"`
+	BuildDate string `json:"buildDate"`
+	GoVersion string `json:"goVersion"`
+}
+
+type RuntimeInfo struct {
+	Birth               time.Time
+	CWD                 string
+	GoroutineCount      int
+	GOMAXPROCS          int
+	GOGC                string
+	GODEBUG             string
+	CorruptionCount     int64
+	ChunkCount          int64
+	TimeSeriesCount     int64
+	LastConfigTime      time.Time
+	ReloadConfigSuccess bool
+	StorageRetention    string
```

Comment:
Keeping this as a string seems a bit odd in an API, on the other hand we already have it in a more structured way in the flags API, so not sure.